components/apimgt/api-store-web/src/site/themes/fancy/templates/api/throttling/template.jag

```diff
@@ -120,10 +120,11 @@ var i, length, app, thumbURL, api = outputs.api, templatesList = outputs.uriTemp
 
 
                                 <% getAuthVal=putAuthVal=postAuthVal=deleteAuthVal=optionAuthVal=""; %>
+                                <% getThrottlingTier=postThrottlingTier=deleteThrottlingTier=optionsThrottlingTier=putThrottlingTier=""; %>
```

Comment:
In JavaScript, variables are functional scoped rather than block scoped. The intention is to clear up all variables into empty on each loop cycle. The other way would be;

getAuthVal="";
putAuthVal="";
postAuthVal="";
deleteAuthVal="";
optionAuthVal="";
getThrottlingTier="";
postThrottlingTier="";
deleteThrottlingTier="";
optionsThrottlingTier="";
putThrottlingTier="";

But It consumes more space in my opinion.